test/lib/linter/js/snapshots/transpiler.ts.md

```diff
@@ -760,7 +760,7 @@ Generated by [AVA](https://avajs.dev).
 
     `// This file contains "sap.ui.define", but no real sap.ui.define call␊
     sap.ui.define;␊
-    //# sourceMappingURL=Noop_NoDefine.ts.map`
+    //# sourceMappingURL=Noop_NoDefine.js.map`
```

Comment:
sourcemap is produced for ".js" instead of ".ts" files. Is his expected and desired?